packages/raylib/raylib.0.3.1/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings for raylib"
+description: "OCaml bindings for raylib"
+maintainer: ["tobiasjammer@gmail.com"]
+authors: ["Tobias Mock"]
+license: "MIT"
+homepage: "https://github.com/tjammer/raylib-ocaml"
+bug-reports: "https://github.com/tjammer/raylib-ocaml/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "dune-configurator"
+  "ctypes" {>= "0.14"}
+  "ppx_cstubs" {>= "0.6"}
+  "conf-mesa" {os = "linux" | os = "freebsd"}
+  "conf-libxcursor" {os = "linux" | os = "freebsd"}
+  "conf-libxi" {os = "linux" | os = "freebsd"}
+  "conf-libxinerama" {os = "linux" | os = "freebsd"}
+  "conf-libxrandr" {os = "linux" | os = "freebsd"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/tjammer/raylib-ocaml.git"
+available: [arch != "arm32" & arch != "ppc64"]
+x-ci-accept-failures: [
+  "centos-7"
+  "oraclelinux-7"
```

Comment:
```suggestion
  "centos-7" # C compiler is too old
  "oraclelinux-7" # C compiler is too old
```
Note that this could probably be fixed upstream by adding `-std=c99` to the CFLAGS